internal/isoutil/isoutil.go

```diff
@@ -230,6 +231,30 @@ func (h *installerHandler) VolumeIdentifier() (string, error) {
 	return strings.TrimSpace(string(volumeId)), nil
 }
 
+func (h *installerHandler) Copy(filePath string) error {
+	srcFile, err := os.OpenFile(h.isoPath, os.O_RDONLY, 0o664)
+	if err != nil {
+		return err
+	}
```

Comment:
Do you need a `defer srcFile.Close()`?

Reply:
It's read-only so no need.

Reply:
Is there some resource that you can point me to that says there is no need to close in this case?

Reply:
Well os.File should be finalized by the garbage collector at least by default: https://golang.org/pkg/runtime/#SetFinalizer
From the reference, the issue with write is that "it would be a mistake to depend on a finalizer to flush an in-memory I/O buffer such as a bufio.Writer, because the buffer would not be flushed at program exit."
Also explained here: https://stackoverflow.com/questions/44667733/should-i-close-log-file-in-go
Any way, to keep consistency and just be on the safe side I guess we can invoke close regardless:)

Reply:
Thank you for the links.